src/pages/[...paths].tsx

```diff
@@ -15,44 +16,59 @@ import React from 'react';
 import TextContent from 'modules/Common/components/TextContent';
 import ThumbGallery from 'modules/Common/components/ThumbGallery';
 import ThumbGalleryItem from 'modules/Common/components/ThumbGalleryItem';
-import { useTranslation } from 'modules/I18n';
+import useTranslation from 'next-translate/useTranslation';
 
-export default function PrivacyPolicyPage({ mdxContent }: WithMdxResult) {
+const FOLDER = 'pages';
```

Comment:
Use a clearer variable name. Okay, it targets a folder. But is this string the `SOURCE_PATH`? The `STATIC_PAGES_PATH` maybe?